app/javascript/packs/home_table_filters.js

```diff
@@ -1,25 +1,42 @@
 // return true if array2 is a subset of array1. Both must be Javascript arrays. Ignores 'hidden' class in second array
 isSubset = function(array1, array2) {
   for (var i = 0; i < array2.length; i++) {
```

Comment:
`isSubset` in javascript could be implemented using javascript `some`.. such as the following filter and some which should filter out the "hidden" and return true if array1 includes any of filtered elements

`array2.filter(e => e !== 'hidden').some((e, i, a) => array1.includes(e))`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some